llvm/docs/CommandGuide/llvm-cxxfilt.rst

```diff
@@ -48,6 +48,10 @@ OPTIONS
 
   Print a summary of command line options.
 
+.. option:: --no-params, -p
+
+  Do not parse function parameters.
```

Comment:
I'd change this to something like "Do not demangle function parameters or return types."